docker/sweepers_driver.py

```diff
@@ -92,6 +92,7 @@
     logging.error(err)
     raise ValueError(f'Failed to parse username/password from PROV_CREDENTIALS value "{provCredentialsStr}": {err}')
 
+log_level = parse_log_level(os.environ.get('LOGLEVEL', 'INFO'))
```

Comment:
I remember that Sean recommended to use a log configuration file to configure the python logs, I don't know how that would integrate in a AWS deployment. We would need to mount the log configuration file in the docker image from a configuration store (I don't know what that is in AWS?). Then we would have the log level in this file.

I don't wnat to overcomplicate the changes right now, so let say it is for comment/discussion, not for change.

Reply:
Can you serve config files or complex objects from the Parameter Store?  Preferably whatever mechanism we use should be trivially-adjustable in AWS (as environment variables are) to allow us to switch into debug logging easily if an issue is found.